framework/include/utils/InputParameters.h

```diff
@@ -996,6 +996,28 @@ InputParameters::setParamHelper<FunctionName, int>(const std::string & /*name*/,
   l_value = oss.str();
 }
 
+template<>
+inline
```

Comment:
Oh you're right about the duplicate symbols, I forgot about that. Are these helpers called outside of the InputParameters class? If not and they can be moved to the .C file, let's do that instead.